eaf.el

```diff
@@ -450,8 +445,8 @@ For now only EAF browser app is supported."
           (apply #'start-process
                  eaf-name
                  eaf-name
-                 eaf-python-command (append (list eaf-python-file) (eaf-get-render-size) (list eaf-http-proxy-host eaf-http-proxy-port eaf-config-dir))
-                 ))
+                 eaf-python-command (append (list eaf-python-file) (eaf-get-render-size)
+                                            (list eaf-http-proxy-host eaf-http-proxy-port (concat user-emacs-directory "eaf/")))))
```

Comment:
(concat user-emacs-directory "eaf")

Don't use eaf/ , / is unix path splitter.